File: AWL/AWL.py
import math
words = input()
letters = 0
res = ""
for word in words:
    if word.isalpha():
        letters += 1
        res += word
#words.count(" ")+1 is the count of words
avg = letters / (words.count(" ")+1)
print(math.ceil(avg),end="")

File: AWL/AWL.java
import java.util.Scanner;

public class Program
{
public static int co(String str)
{
    int totalsize = str.length();
    int word = 0;
    int letters = 0;
    for(int x =0;x <totalsize;x++)
    {
   if(str.charAt(x) == ' ')
        word++;
else if(Character.isLetter(str.charAt(x)))
        letters++;
    }
  double avg = Double.valueOf(letters)/(word+1);
  int res = (int)Math.ceil(avg);
  return res;
}
 public static void main(String[] args)
  {
     Scanner in = new Scanner(System.in);
     System.out.print(co(in.nextLine()));
  }
}
